package.json

```diff
@@ -1,24 +1,32 @@
 {
   "name": "front-end-dev-test",
-  "version": "0.0.0",
-  "description": "",
+  "version": "0.0.1",
+  "description": "A blocky puzzle for software engineers.",
   "main": "index.js",
   "scripts": {
-    "start": "gulp",
-    "test": "gulp buildtests && open test/index.html"
+    "build": "webpack --config webpack.config.js",
+    "start": "webpack-dev-server --open",
+    "test": "mocha --require babel-polyfill --compilers js:babel-register,jsdom-global/register 'app/**/*.test.js'",
+    "test:watch": "yarn test -- --watch",
+    "prettier": "prettier --single-quote --write 'app/**/*.js'"
   },
   "devDependencies": {
-    "babel-preset-es2015": "^6.1.18",
-    "babelify": "^7.2.0",
-    "browserify": "^13.0.0",
+    "babel-core": "^6.24.1",
+    "babel-loader": "^6.4.1",
+    "babel-polyfill": "^6.23.0",
+    "babel-preset-env": "^1.3.3",
+    "babel-register": "^6.24.1",
     "chai": "^3.4.1",
-    "gulp": "^3.8.11",
-    "gulp-babel": "^6.1.0",
-    "gulp-browserify": "^0.5.1",
-    "gulp-connect": "^2.2.0",
-    "gulp-less": "^3.0.1",
-    "vinyl-buffer": "^1.0.0",
-    "vinyl-source-stream": "^1.1.0",
-    "watchify": "^3.7.0"
-  }
-}
+    "css-loader": "^0.28.0",
+    "jsdom": "^9.12.0",
+    "jsdom-global": "^2.1.1",
+    "less": "^2.7.2",
+    "less-loader": "^4.0.3",
+    "mocha": "^3.2.0",
+    "prettier": "^0.22.0",
```

Comment:
It doesn't do anything with it by itself and the docs do not say anything about the `package.json` script. Allows us to do `yarn prettier` and see if anything changes! (so if they found it themselves). Could remove